### PR TITLE
fix(helpers): pass null if options.channelId is falsy

### PR DESCRIPTION
**Pull request by rigormorrtiss**
image

### CODE REVIEW

**Review by itohatweb — CHANGES_REQUESTED** [review]
This is a bad solution. With that one you require the user to always pass a channelId if the bot shall not move the user.

**Comment by Skillz4Killz** [comment]
We have `disconnectMember()` no?

**Comment by golfIII** [comment]
> 
> 
> We have `disconnectMember()` no?

Hi, I'm the Discord member in the image above.

Same issue occurs when using `disconnectMember()`. Perms are valid, channel is valid, and the member being kicked is valid and is successfully returned from both `channel.disconnect()` and `disconnectMember()`.

I believe @ayntee was able to reproduce the issue on his end as well.

**Comment by ayntee** [comment]
@Skillz4Killz disconnectMember() calls editMember() under the hood

**Comment by ayntee** [comment]
> This is a bad solution. With that one you require the user to always pass a channelId if the bot shall not move the user.

@itohatweb looks like the PR has been updated

File: src/helpers/members/edit_member.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export async function editMember(guildId: bigint, memberId: bigint, options: Mod
     endpoints.GUILD_MEMBER(guildId, memberId),
     snakelize({
       ...options,
-      channelId: options.channelId ? bigintToSnowflake(options.channelId) : undefined,
+      channelId: typeof options.channelId === "bigint" ? bigintToSnowflake(options.channelId) : options.channelId,
```
Comment (Skillz4Killz) [inline src/helpers/members/edit_member.ts:57]:
```suggestion
      channelId: options.channelId ? bigintToSnowflake(options.channelId) : options.channelId,
```

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**